File: src/java/section2/FizzBuzz.java
import java.util.Scanner;

public class FizzBuzz {
  public static void main(String args[]){
    System.out.print("num: ");
    Scanner scan = new Scanner(System.in);
    int num = scan.nextInt();
    scan.close();

    for(int i=1;i<=num;i++){
      if(i%15 == 0){
        System.out.println(i+": FizzBuzz");
      }else if(i%3 == 0){
        System.out.println(i+": Fizz");
      }else if(i%5 == 0){
        System.out.println(i+": Buzz");
      }else{
        System.out.println(i);
      }
    }
  }
}

File: src/cpp/section2/FizzBuzz.cpp
#include<iostream>
using namespace std;

int main(void){
  int num;

  cout << "num: ";
  cin >> num;

  for(int i=1;i<=num;i++){
    if(i%15 == 0){
      cout << i << ": FizzBuzz" << endl;
    }else if(i%3 ==0){
      cout << i << ": Fizz" << endl;
    }else if(i%5 == 0){
      cout << i << ": Buzz" << endl;
    }else{
      cout << i << endl;
    }
  }

  return 0;
}